webapp/src/main/java/com/box/l10n/mojito/metrics/NamingConventionConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.box.l10n.mojito.metrics;
+
+import com.google.common.base.Strings;
+
+import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import io.micrometer.core.instrument.config.NamingConvention;
+
+@Component
+@ConfigurationPropertiesBinding
```

Comment:
`@ConfigurationPropertiesBinding` allows Spring to use this Converter for configuration parameters. It'll be used for `NamingConvention` within `StatsdNamingConventionConfig`